modules/customizing-the-web-console-URL.adoc

```diff
@@ -97,3 +97,7 @@ spec:
 <2> Reference to a secret in the `openshift-config` namespace that contains a TLS certificate (`tls.crt`) and key (`tls.key`). This is required if the domain for the custom hostname suffix does not match the cluster domain suffix. The secret is optional if the suffix matches.
 
 . Save the file to apply the changes.
+
+== Recognizing resource/project limits and quotas
```

Comment:
We should probably find a way to avoid using the `/` in this, as per IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes

Reply:
@nalhadef I'm not seeing this addressed. This should be rewritten to avoid the `/`.

Reply:
fixed.
![image](https://user-images.githubusercontent.com/83772007/231259916-511ddd54-b195-4aeb-8c07-82f8096923e9.png)